pkg/client/client.go

```diff
@@ -1,6 +1,7 @@
 package client
 
 import (
+	clientconfigv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
```

Comment:
why you don't just add the openshift API to the scheme?

https://github.com/openshift/api/blob/master/config/install.go#L18